docs/user-guide/images.md

```diff
@@ -83,6 +86,9 @@ The kubelet will fetch and periodically refresh ECR credentials.  It needs the f
 **Note:** if you are running on Google Container Engine (GKE), there will already be a `.dockercfg` on each node
 with credentials for Google Container Registry.  You cannot use this approach.
 
+**Note:** if you are running on AWS EC2 and are using the EC2 Container Registry (ECR), the kubelet on each node will
```

Comment:
I would reverse the wording a bit: "if you are running (x), you can not use this approach. The kubelete on each node will (y)." 

"Cannot" isn't a word, btw. :(

:)